From a192682d268a73b21f4803274228270d015fd226 Mon Sep 17 00:00:00 2001
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Date: Wed, 6 Nov 2024 17:22:04 +0800
Subject: [PATCH] set -DFETCHCONTENT_FULLY_DISCONNECTED:BOOL=OFF

---
 debian/rules.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules.em b/debian/rules.em
index a856369d..9e15949a 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -35,6 +35,7 @@ override_dh_auto_configure:
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DAMENT_PREFIX_PATH="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DFETCHCONTENT_FULLY_DISCONNECTED:BOOL=OFF  \
 		$(BUILD_TESTING_ARG)
 
 override_dh_auto_build:
-- 
2.48.1

