--- a/detect_aruco_markers_behavior/src/detect_aruco_markers_behavior.cpp
+++ b/detect_aruco_markers_behavior/src/detect_aruco_markers_behavior.cpp
@@ -191,7 +191,7 @@
       cv::composeRT(rvec * 0, tvec * 0, rvec, tvec, rout, tout);
       aruco_position = tout;
       aruco_rotation = rout;
-      cv::aruco::drawAxis(output_image, camera_matrix_, dist_coeffs_, rout, tout, 0.08625);
+      cv::drawFrameAxes(output_image, camera_matrix_, dist_coeffs_, rout, tout, 0.08625, 3);
     }
   }
 
